src/infrastructure/database/modules/sqlite/utils/sqlite-pragma.manager.ts

```
@@ -0,0 +1,287 @@
+      appliedSettings: {},
+      failedSettings: [],
+      criticalSettingsApplied: true,
+    };
+
+    // Build PRAGMA settings from configuration
+    const pragmaSettings = this.buildPragmaSettings(config);
+
+    // Apply each PRAGMA setting
+    for (const setting of pragmaSettings) {
+      try {
+        this.applySinglePragma(client, setting);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)